clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/EventualServerStoreProxy.java

```diff
@@ -126,5 +178,13 @@ public void replaceAtHead(long key, Chain expect, Chain update) {
   @Override
   public void clear() throws TimeoutException {
     delegate.clear();
+    clearInProgressCounter.incrementAndGet();
+  }
+
+  private void incrementInvalidationsCounter(long key) {
+    AtomicInteger counter = new AtomicInteger(1);
+    if ((invalidationsInProgress.putIfAbsent(key, counter)) != null) {
```

Comment:
but the null check is there for the cases when key is was absent, so null will be returned leading to a nullpointer , right?